packages/chronicle/src/themes/paper/Layout.module.css

```
@@ -0,0 +1,32 @@
+.layout {
+  --paper-sidebar-width: 260px;
+  min-height: 100vh;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)